From e27ffc6bfd31e2bd4085a73842b3f6397a7e9f56 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Yilli=C3=A9?= <yilliee@protonmail.com>
Date: Fri, 24 Dec 2021 09:30:21 +0000
Subject: [PATCH 1/2] config : Don't enable adb auth on non-eng builds

---
 config/common.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/common.mk b/config/common.mk
index da899b2..fb8441e 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -18,7 +18,7 @@ ifeq ($(TARGET_BUILD_VARIANT),eng)
 PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=0
 else
 # Enable ADB authentication
-PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=1
+#PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=1
 
 # Enable one-handed mode
 PRODUCT_PRODUCT_PROPERTIES += \
-- 
2.17.1

