From 784fe71eab18c0c27d27743654db3c7014f41068 Mon Sep 17 00:00:00 2001
From: Yillie <m.taha6f@gmail.com>
Date: Tue, 4 May 2021 19:47:05 +0500
Subject: [PATCH 1/6] Create Octavi.mk using havoc.mk as a reference

---
 Octavi.mk | 3 +++
 1 file changed, 3 insertions(+)
 create mode 100644 Octavi.mk

diff --git a/Octavi.mk b/Octavi.mk
new file mode 100644
index 0000000..241ea78
--- /dev/null
+++ b/Octavi.mk
@@ -0,0 +1,3 @@
+$(call inherit-product, vendor/octavi/config/common_full_phone.mk)
+$(call inherit-product, device/octavi/sepolicy/common/sepolicy.mk)
+-include vendor/octavi/build/core/config.mk
-- 
2.19.0

