From c2c5290b8b37f1f0c620ca032d52bef874f6c5f9 Mon Sep 17 00:00:00 2001
From: Pierre-Hugues Husson <phh@phh.me>
Date: Tue, 3 Dec 2019 14:04:17 +0100
Subject: [PATCH 10/22] Fix for some Huawei camera

---
 services/camera/libcameraservice/CameraService.cpp | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/services/camera/libcameraservice/CameraService.cpp b/services/camera/libcameraservice/CameraService.cpp
index f6bf7e6f68..3a1d9890b2 100644
--- a/services/camera/libcameraservice/CameraService.cpp
+++ b/services/camera/libcameraservice/CameraService.cpp
@@ -16,7 +16,7 @@
 
 #define LOG_TAG "CameraService"
 #define ATRACE_TAG ATRACE_TAG_CAMERA
-//#define LOG_NDEBUG 0
+#define LOG_NDEBUG 0
 
 #include <algorithm>
 #include <climits>
@@ -240,10 +240,11 @@ status_t CameraService::enumerateProviders() {
             if (!cameraFound) {
                 addStates(id8);
             }
-        }
-
-        if (getCameraState(id8) == nullptr) {
             onDeviceStatusChanged(id8, CameraDeviceStatus::PRESENT);
+        } else {
+            if (getCameraState(id8) == nullptr) {
+                onDeviceStatusChanged(id8, CameraDeviceStatus::PRESENT);
+            }
         }
     }
 
-- 
2.33.0

